lib/fwupd_device_flag_l10n.dart

```diff
@@ -0,0 +1,106 @@
+      case FwupdDeviceFlag.notified:
+        return l10n.fwupdDeviceFlagNotified;
+      case FwupdDeviceFlag.useRuntimeVersion:
+        return null;
+      case FwupdDeviceFlag.installParentFirst:
+        return l10n.fwupdDeviceFlagInstallParentFirst;
+      case FwupdDeviceFlag.isBootloader:
+        return l10n.fwupdDeviceFlagIsBootloader;
+      case FwupdDeviceFlag.waitForReplug:
+        return l10n.fwupdDeviceFlagWaitForReplug;
+      case FwupdDeviceFlag.ignoreValidation:
+        return l10n.fwupdDeviceFlagIgnoreValidation;
+      case FwupdDeviceFlag.trusted:
+        return l10n.fwupdDeviceFlagTrusted;
+      case FwupdDeviceFlag.anotherWriteRequired:
+        return null;
+      case FwupdDeviceFlag.noAutoInstanceIds:
+        return null;
+      case FwupdDeviceFlag.needsActivation:
+        return l10n.fwupdDeviceFlagNeedsActivation;
+      case FwupdDeviceFlag.ensureSemver:
+        return null;
+      case FwupdDeviceFlag.historical:
+        return null;
+      case FwupdDeviceFlag.onlySupported:
+        return null;
+      case FwupdDeviceFlag.willDisappear:
+        return l10n.fwupdDeviceFlagWillDisappear;
+      case FwupdDeviceFlag.canVerify:
+        return l10n.fwupdDeviceFlagCanVerify;
+      case FwupdDeviceFlag.canVerifyImage:
+        return null;
+      case FwupdDeviceFlag.dualImage:
+        return l10n.fwupdDeviceFlagDualImage;
+      case FwupdDeviceFlag.selfRecovery:
+        return l10n.fwupdDeviceFlagSelfRecovery;
+      case FwupdDeviceFlag.usableDuringUpdate:
+        return l10n.fwupdDeviceFlagUsableDuringUpdate;
+      case FwupdDeviceFlag.versionCheckRequired:
+        return l10n.fwupdDeviceFlagVersionCheckRequired;
+      case FwupdDeviceFlag.installAllReleases:
+        return l10n.fwupdDeviceFlagInstallAllReleases;
+      case FwupdDeviceFlag.mdSetName:
+        return null;
+      case FwupdDeviceFlag.mdSetNameCategory:
+        return null;
+      case FwupdDeviceFlag.mdSetVerfmt:
+        return null;
+      case FwupdDeviceFlag.mdSetIcon:
+        return null;
+      case FwupdDeviceFlag.addCounterpartGuids:
+        return null;
+      case FwupdDeviceFlag.noGuidMatching:
+        return null;
+      case FwupdDeviceFlag.updatableHidden:
+        return l10n.fwupdDeviceFlagUpdatable;
+      case FwupdDeviceFlag.skipsRestart:
+        return null;
+      case FwupdDeviceFlag.hasMultipleBranches:
+        return l10n.fwupdDeviceFlagHasMultipleBranches;
+      case FwupdDeviceFlag.backupBeforeInstall:
+        return l10n.fwupdDeviceFlagBackupBeforeInstall;
+      case FwupdDeviceFlag.wildcardInstall:
+        return l10n.fwupdDeviceFlagWildcardInstall;
+      case FwupdDeviceFlag.onlyVersionUpgrade:
+        return l10n.fwupdDeviceFlagOnlyVersionUpgrade;
+      case FwupdDeviceFlag.unreachable:
+        return l10n.fwupdDeviceFlagUnreachable;
+      case FwupdDeviceFlag.affectsFde:
+        return l10n.fwupdDeviceFlagAffectsFde;
+      default:
+        return 'missing: ${describeEnum(this)}';
```

Comment:
is this necessary because `missing_enum_constant_in_switch` would fail?

Reply:
No, it's not anymore - all enums are taken care of :)